### ISSUE TITLE
Code for Segmentation

### ISSUE THREAD

**Issue by eugenelawrence** [zhanghang1989/PyTorch-Encoding#30]
When will the code for segmentation be released?

**Comment by zhanghang1989** [comment]
Thx for interest in this work. I am a little busy these days. I will clean the code and release it ASAP.

**Comment by acgtyrant** [comment]
Looking forward to it too.

**Comment by zhanghang1989** [comment]
This issue has been resolved in favor of releasing PyTorch compatible Synchronized Cross-GPU [encoding.nn.BatchNorm2d](http://hangzh.com/PyTorch-Encoding/syncbn.html) and the [example](https://github.com/zhanghang1989/PyTorch-SyncBatchNorm).

**Comment by eugenelawrence** [comment]
@zhanghang1989 Sorry, I am requiring for the demo of the semantic segmentation experiments, rather than the code for synchronized cross-gpu.

**Comment by zhanghang1989** [comment]
Thanks folks! The code is broken, due to the recent updates of PyTorch. I am working on it, will release it ASAP.

**Comment by devkicks** [comment]
Hi @zhanghang1989 

Any updates on when this will be available?
